MODULE_GUIDELINES.md

```diff
@@ -49,7 +49,7 @@ A detailed explanation of the PR workflow can be seen here: https://github.com/a
 
 # Extras maintainers list
 
-The full list of maintainers for modules is located here: https://github.com/ansible/ansibullbot/blob/master/MAINTAINERS.txt
+The full list of maintainers for modules is located here: https://github.com/ansible/ansible/blob/devel/.github/BOTMETA.yml
```

Comment:
Ok, so this is no longer valid with BOTMETA.yml. It now is using the `author` information from the modules files as the source, and extends this with the information from BOTMETA.yml. So that should probably be reflected in this paragraph.

The title "Extras maintainers list" also seems legacy, there's no Extras anymore.